Documentation/ApiOverview/ContentElements/ContentElementsWizard.rst

```diff
@@ -40,7 +37,8 @@ plugin is `registration`.
           }
       }
 
-   You may want to replace title and description, using language files for translation, for example:
+   You may want to replace :typoscript:`title` and :typoscript:`description`,
+   using language files for translation, for example:
 
    .. code-block:: typoscript
```

Comment:
Shall we also write a caption with the filename here for clarity?

Reply:
I want to avoid it as the main context (mod....) is missing. I added a "from above" for more clarity.